### PR TITLE
fix(appeals): pptx upload

### PR DESCRIPTION
**Pull request by fabfab-kc**
Fixes the validation of PPTX files
Adds an overrideable default for no input in date function
Align usage of some date mappers

## Type of change 🧩

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: appeals/web/src/server/lib/mappers/appeal/submappers/appellant-statement-due-date.mapper.js [hunk 1]
```diff
@@ -10,8 +10,10 @@ export const mapAppellantStatementDueDate = ({
 	textSummaryListItem({
 		id: 'appellant-statement-due-date',
 		text: 'Appellant statement due',
-		value:
-			dateISOStringToDisplayDate(appealDetails.appealTimetable?.appellantStatementDueDate) || '',
+		value: dateISOStringToDisplayDate(
+			appealDetails.appealTimetable?.appellantStatementDueDate,
+			'Due date not yet set'
+		),
```
Comment (Si3rr4wow) [inline appeals/web/src/server/lib/mappers/appeal/submappers/appellant-statement-due-date.mapper.js:16]: Did you mean to add this default here?

File: appeals/web/src/server/lib/mappers/appeal/submappers/complete-date.mapper.js [hunk 2]
```diff
@@ -6,9 +6,7 @@ export const mapCompleteDate = ({ appealDetails, currentRoute, userHasUpdateCase
 	textSummaryListItem({
 		id: 'complete-date',
 		text: 'Complete',
-		value:
-			dateISOStringToDisplayDate(appealDetails.appealTimetable?.completeDate) ||
-			'Due date not yet set',
+		value: dateISOStringToDisplayDate(appealDetails.appealTimetable?.completeDate),
```
Comment (Si3rr4wow) [inline appeals/web/src/server/lib/mappers/appeal/submappers/complete-date.mapper.js:9]: Di you mean to remove this default?

File: packages/appeals/constants/documents.js [hunk 3]
```diff
@@ -56,7 +56,7 @@ export const VALID_MIME_TYPES = Object.freeze({
 	},
 	'application/vnd.ms-powerpoint': { hexSignature: 'D0CF11E0', extensions: ['ppt'] },
 	'application/vnd.openxmlformats-officedocument.presentationml.presentation': {
-		hexSignature: 'D0CF11E0',
+		hexSignature: '504B03041400',
```
Comment (Si3rr4wow) [inline packages/appeals/constants/documents.js:59]: This feels like even more forbidden knowledge than Regex. Is there a reference table for there somewhere online?

File: appeals/web/src/server/lib/mappers/appeal/submappers/appellant-statement-due-date.mapper.js (hunk 1 above)
Comment (fabfab-kc) [reply appeals/web/src/server/lib/mappers/appeal/submappers/appellant-statement-due-date.mapper.js]: Well spotted

File: packages/appeals/constants/documents.js (hunk 3 above)
Comment (fabfab-kc) [reply packages/appeals/constants/documents.js:59]: Google! however this was a bit obvious as all new office docs share the same

File: appeals/web/src/server/lib/mappers/appeal/submappers/complete-date.mapper.js (hunk 2 above)
Comment (fabfab-kc) [reply appeals/web/src/server/lib/mappers/appeal/submappers/complete-date.mapper.js:9]: yes. Agreed with Rachel we need a review and alignment of all defaults - should be all blank at the moment

**Review by Si3rr4wow — APPROVED** [review]

**Pull request merged**